content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -43,55 +40,138 @@ Istio control plane, Envoy can then communicate with the **single**
+
+> Please wait for the Istio control plane to finish initializing
+before proceeding to steps in this section.
+
+> These operations must be run on the Istio control plane cluster
+to capture the Istio control-plane service endpoints--e.g. Pilot, Policy,
+and Statsd Pod IP endpoints.
+
+> If Helm is used with Tiller on each remote, copy the environment
+variables to each node before using Helm to connect the remote
+cluster to the Istio control plane.
+
+{{< text bash >}}
+$ export PILOT_POD_IP=$(kubectl -n istio-system get pod -l istio=pilot -o jsonpath='{.items[0].status.podIP}')
+$ export POLICY_POD_IP=$(kubectl -n istio-system get pod -l istio=mixer -o jsonpath='{.items[0].status.podIP}')
+$ export STATSD_POD_IP=$(kubectl -n istio-system get pod -l istio=statsd-prom-bridge -o jsonpath='{.items[0].status.podIP}')
+$ export TELEMETRY_POD_IP=$(kubectl -n istio-system get pod -l istio-mixer-type=telemetry -o jsonpath='{.items[0].status.podIP}')
+$ export ZIPKIN_POD_IP=$(kubectl -n istio-system get pod -l app=jaeger -o jsonpath='{.items[0].status.podIP}')
+{{< /text >}}
+
+Proceed to one of the options for connecting the remote cluster to the local cluster:
+
+* [via kubectl with helm](#use-kubectl-with-helm-to-connect-the-remote-cluster-to-the-local)
+
+* [via helm plus tiller](#alternatively-use-helm-and-tiller-to-connect-the-remote-cluster-to-the-local)
+
+{{< idea_icon >}}
+The default behavior is to enable automatic sidecar injection on the remote clusters.  For manual sidecar injection refer to the [manual sidecar example](#remote-cluster-manual-sidecar-injection-example)
+
+### Use kubectl with Helm to connect the remote cluster to the local
+
+1.  Use the helm template command on a remote to specify the Istio control plane service endpoints:
 
     {{< text bash >}}
-    $ cat <<EOF | kubectl create -f -
-    kind: ClusterRole
-    apiVersion: rbac.authorization.k8s.io/v1
-    metadata:
-      name: istio-reader
-    rules:
-       - apiGroups: ['']
-         resources: ['nodes', 'pods', 'services', 'endpoints']
-         verbs: ['get', 'watch', 'list']
-    EOF
+    $ helm template install/kubernetes/helm/istio-remote --namespace istio-system --name istio-remote --set global.remotePilotAddress=${PILOT_POD_IP} --set global.remotePolicyAddress=${POLICY_POD_IP} --set global.remoteTelemetryAddress=${TELEMETRY_POD_IP} --set global.proxy.envoyStatsd.enabled=true --set global.proxy.envoyStatsd.host=${STATSD_POD_IP} --set global.remoteZipkinAddress=${ZIPKIN_POD_IP} > $HOME/istio-remote.yaml
+    {{< /text >}}
+
+1.  Create a namespace for remote Istio.
+
+    {{< text bash >}}
+    $ kubectl create ns istio-system
+    {{< /text >}}
+
+1.  Instantiate the remote cluster's connection to the Istio control plane:
+
+    {{< text bash >}}
+    $ kubectl create -f $HOME/istio-remote.yaml
+    {{< /text >}}
+
+1.  Label the remote cluster's `default` namespace for auto-sidecar injection.
```

Comment:
In the event that an application isn't deployed in the default namespace, this won't work.  Can you rework this small section to avoid confusion here?

Reply:
Should I just add "Example:" in front like "Example:  Label the ..." here?  I have a little repeat for all namespaces blurb right below the bash block that adds clarification.

Reply:
I'll change it to "Label all the remote cluster's namespaces requiring auto-sidecar injection.  The following example labels the `default` namespace"